mhkit/dolfyn/velocity.py

```diff
@@ -900,7 +675,8 @@ def turbulent_kinetic_energy(self, veldat, noise=[0, 0, 0], detrend=True):
 
         """
         if 'dir' in veldat.dims:
-            vel = veldat[:3].values
+            # will error for ADCP 4-beam, but not for single beam
+            vel = veldat.values
```

Comment:
Is there an alternative that won't work break in the ADCP 4-beam case?

Reply:
Ah, it should error for ADCP's, and I should add an exception here